### DISCUSSION TITLE
How to import/export RTF file content to the editor?

### DISCUSSION THREAD

**Discussion by KiddoV** [slab/quill#4197] [Q&A]
How do I import ``rtf`` file content into the editor?
For example:
```rtf
{\rtf1\ansi\ansicpg1252\deff0\nouicompat{\fonttbl{\f0\fnil\fcharset0 Calibri;}}
{\colortbl ;\red155\green0\blue211;}
{\*\generator Riched20 10.0.19041}\viewkind4\uc1 
\pard\sa200\sl276\slmult1\cf1\b\f0\fs22\lang9 Testing RTF\cf0\b0\line This is an RTF content...\line\i Hello\i0\par
}
```
After editing, how do I export the content out as ``rtf`` raw string?

**Comment by luin** [comment]
I guess you will need some way to parse RTF content to html/plain text in order to import them to Quill since Quill doesn't provide a built-in way.
